changelog/pending/20240222--backend-diy--remove-warning-about-non-project-mode-depracation.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: chore
+  scope: backend/diy
+  description: Remove warning about non-project mode depracation.
```

```suggestion
  description: Remove warning about non-project mode deprecation.
```

Do we even need to call this out in the changelog?

---

We called it out in the changelog originally.